docs/marketplace/dev/front-end/ui-components-library/data-transformers/lens.md

```diff
@@ -0,0 +1,77 @@
+---
+title: Data Transformer Lens
+description: This document provides details about the Data Transformer Lens service in the Components Library.
+template: concept-topic-template
+---
+
+
+This document provides details about the Data Transformer Lens service in the Components Library.
+
+## Overview
+
+Data Transformer Lens is an Angular Service that updates the nested object by path using another Data Transformer set up with a configuration object.
+In the example below date in `col2` will be transformed.
+
+```ts
+<spy-table
+  [config]="{
```

Comment:
Please do not use table in examples as it's more complicated.
Better to use `spy-select` with just the datasource and transformer example.